cli/flags.rs

```diff
@@ -1398,6 +1405,10 @@ fn permission_args<'a, 'b>(app: App<'a, 'b>) -> App<'a, 'b> {
     .arg(
       Arg::with_name("allow-run")
         .long("allow-run")
+        .min_values(0)
+        .takes_value(true)
+        .use_delimiter(true)
+        .require_equals(true)
```

Comment:
I think some kind of validator is needed here, similar to `--allow-env` PR

Reply:
sure, but how would we validate passed commands? check if they actually exist?